website/index.php

```diff
@@ -116,9 +116,10 @@ class="w-75"></span><span class="w-50"></span></button>
 
                                 <dl>
                                     <?php foreach ($line as $key => $value): ?>
+                                        <?php if ($key === "borderColor" && $value === "") continue; ?>
```

Comment:
I think, shortOperatorName and lineName can also be skipped.

Reply:
At this point, we could maybe also move to a white-list of properties that we want to show.

Reply:
Makes also sense.

But shortOperatorName is the same in every section as the heading and the lineName is rendered in the icon :D